pkg/test-helpers/mock.go

```diff
@@ -0,0 +1,55 @@
+package test_helpers
```

Comment:
`test_helpers` is a really generic name and has the potential to get really big the more things we move into here. Why not put this in `pkg/mock`?

Reply:
I get a circular import unfortunately.

Reply:
try to fix it another way?

Reply:
I can move it to `mock`, just not `pkg/mock`.